src/freerct.cpp

```diff
@@ -52,8 +53,14 @@ static void PrintUsage()
 {
 	printf("Usage: freerct [options]\n");
 	printf("Options:\n");
-	printf("  -h, --help           Display this help text and exit\n");
-	printf("  -l, --load [file]    Load game from specified file\n");
+	printf("  -h, --help           Display this help text and exit.\n");
+	printf("  -l, --load [file]    Load game from specified file.\n");
+	printf("  -a, --language lang  Use the specified language.\n");
+	printf("                       Valid languages are:");
+	for (int i = 0; i < LANGUAGE_COUNT; ++i) {
+		printf(" %s", _lang_names[i]);
+	}
```

Comment:
The list of languages can be separate from the `-a` option. (Just add an empty line, then print the text starting from the left margin.

If you want to be fancy, go to the next line after 70 or so characters :)